Documentation/Contributors/BuildGuide/README.md

```diff
@@ -120,39 +131,41 @@ npm run [target-name]
+    - `--removePragmas` - Optimizes the output by removing debugging code that validates function input and throws `DeveloperError`s. The removed sections are marked with `//>>includeStart('debug', pragmas.debug);` blocks in the code.
+    - `--node` - Bundles an `index.cjs` module targeted for use in NodeJS
+  - `build-watch` - A never-ending task that watches your file system for changes to Cesium and builds the source code as needed. All `build` options are also available for this task.
+  - `build-apps` - Builds the example applications (such as Cesium Viewer) to produce self-contained, minified, deployable versions in the `Build` directory.
+  - `build-doc` - Generates HTML documentation in `Build/Documentation` using [JSDoc 3](https://github.com/jsdoc3/jsdoc). More [details here](https://github.com/rahwang/cesium/tree/main/Documentation/Contributors/DocumentationGuide).
+  - `build-ts` - Generates a TypeScript definitions file for the Cesium library
+  - `build-third-party` - Generates `ThirdParty.json`, a file which lists the latest licensing information of installed third party modules
   - `release` - A full release build that creates a shippable product, including generating documentation.
-  - `makeZipFile` - Builds a zip file containing all release files. This includes the source tree (suitable for use from an AMD-aware application), plus the combined and minified Cesium.js files, the generated documentation, the test suite, and the example applications (in both built and source form).
+  - `make-zip` - Builds a zip file containing all release files. This includes the source ESM modules, bundled ESM and IIFE format `Cesium.js`, plus the bundled minified versions of ESM and IIFE, the generated documentation, the test suite, and the example applications (in both built and source form).
 - **Utility scripts** -- code coverage, static code analysis, and other utilities
-  - `coverage` - Runs coverage and opens the default browser with the results.
-  - `eslint` - Runs [ESLint](http://eslint.org/), a static code analysis tool, on the entire source tree.
-  - `eslint-watch` - A never-ending task that watches your file system for changes to Cesium and runs ESLint on any changed source files.
-  - `clean` - Removes all generated build artifacts.
+  - `clean` - Removes all generated build artifacts
   - `cloc` - Runs [CLOC](https://github.com/AlDanial/cloc) to count the lines of code on the Source and Specs directories. This requires [Perl](http://www.perl.org/) to execute.
+  - `coverage` - Runs coverage and opens the default browser with the results
+  - `eslint` - Runs [ESLint](http://eslint.org/), a static code analysis tool, on the entire source tree
+  - `eslint-watch` - A never-ending task that watches your file system for changes to Cesium and runs ESLint on any changed source files
```

Comment:
This isn't a valid command in this branch. In VSCode, we use the extension, but I suppose something like this would be helpful for people who prefer coding in other tools.